src/ed_optimization/application/features/order/handlers/commands/process_order_command_handler.py

```
@@ -1,15 +1,15 @@
 
-from ed_optimization.application.common.responses.base_response import BaseResponse
-from ed_optimization.application.contracts.infrastructure.message_queue.abc_subscriber import (
-    ABCSubscriber,
-)
-from ed_optimization.application.contracts.infrastructure.persistence.abc_unit_of_work import (
-    ABCUnitOfWork,
-)
-from ed_optimization.application.features.order.requests.commands import ProcessOrderCommand
+from ed_optimization.application.common.responses.base_response import \
+    BaseResponse
+from ed_optimization.application.features.order.handlers.commands import orders
```

Ruff (0.8.2)

`ed_optimization.application.features.order.handlers.commands.orders` imported but unused

Remove unused import: `ed_optimization.application.features.order.handlers.commands.orders`

(F401)